### DISCUSSION TITLE
Dúvidas sobre portfólio e habilidades

### DISCUSSION THREAD

**Discussion by solilorys** [frontendbr/forum#2339] [Perguntas e Respostas] [answered]
Olá pessoal!

Tenho uma dúvida sobre portfólios e habilidades.
Como sou uma iniciante no ramo de desenvolvimento frontend, eu estava me perguntando se vale a pena desenvolver um projeto (portfolio) somente com coisas puras (HTML, CSS E JS) sem utilizar JQuery, Bootstrap e/ou outras opções, pois assim eu demonstraria que tenho conhecimento na base. Entretanto as ferramentas como citei (Jquery...) são necessárias no mercado e muito legais e interessantes de se usar em um projeto portfólio, mas não sei o que pode pesar mais

O portfólio puro ou um portfólio com várias tecnologias além do puro? Alguém pode iluminar meu caminho com alguma dica e/ou conselho?

**Comment by DiogoCercilio** [comment]
Eu sugiro vc desenvolver o básico... e depois ir melhorando. O que é mais fácil pra vc? Usar essas ferramentas ou não?

**Comment by felquis** [comment, marked as answer]
Neste link tem alguns portfólios https://gist.github.com/felquis/4060562 que compartilharam comigo pode te servir de alguma inspiração.

Desde antigamente, já diziam que os medidores de habilidades não é lá um conteúdo muito rico, pois o nível informado é muito subjetivo embora pensar em uma auto-avaliação é um exercício interessante para si mesmo.

Se seu objetivo é chamar atenção, e se tornar conhecida nos fóruns, grupos, basta você começar a questionar todas as boas práticas "estabelecidas" do mercado e fazer perguntas como "O figma vai substituir o CSS?" onde muita gente tem opinião, mas nenhuma opinião leva a lugar nenhum kkkkkk eu mesmo apareço nos fóruns por que quem tenta ajudar, assim como quem faz perguntas, também pode acabar incrementando o networking, e quem sabe até trabalhando junto em algo no futuro.

Um conteúdo mais rico, é por exemplo você compartilhar um sonho seu, um desejo, uma ambição, sua história, seu processo de estudo, algumas anotação de um estudo que fez, um evento/meetup que organizou ou participou - tudo que tenha sinergia com o tema profissional e vá de encontro com seu objetivo final.
